stp/constraints.lisp

```diff
@@ -46,7 +46,7 @@
 ;;    2.3 heartbeat    ;;
 ;;                     ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;
-(defconstraint first-row (:domain {0})
+(defconstraint first-row (:domain {0}) ;; ""
```

Comment:
What is the "" ?

Reply:
Never mind them, I sometimes add them to help my IDE with the colors ...